Makefile

```diff
@@ -81,6 +81,7 @@ fmt-js:
 
 fmt-python:
 	$(MAKE) -C languages/python fmt
+	black languages/python
```

Comment:
I don't know why we need this addition to the makefile, the sub makes should already run `black`.

@Anniepoo was seeing that test files were not properly reformatted

Reply:
However, black does run for test files.

```
(venv) ➜  oso git:(main) ✗ black -v .
... [snip] ...
/Users/dhatch/code/oso/languages/python/oso/tests/test_partial.py wasn't modified on disk since last run.
/Users/dhatch/code/oso/languages/python/oso/tests/test_polar.py wasn't modified on disk since last run.
```

Reply:
Yeah, was just checking the same thing

Reply:
so do we need this or not?

Reply:
I honestly can't tell.

Reply:
My PR was failing because one line in a python test was getting reformatted by black - strange part is, after I manually black'ed that directory, it's still failing, so as of this moment I'm baffled.

Reply:
Maybe you have a different version of `black` than the one used in CI?

Reply:
Either way, it seems like we should remove this line

Reply:
Going to drop this for now, we aren't entirely sure what caused black to not run on the test file on Annie's machine. If we see another formatting issue we can revisit and maybe just replace all the sub make calls with a top level `black languages/python`.